### PR TITLE
Rep 808 valkyrie

### PR DESCRIPTION
**Pull request by tylerroyal**

### CODE REVIEW

File: repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/filters/valkyrie/system-model.cfg.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<system-model xmlns="http://docs.openrepose.org/repose/system-model/v2.0">
+  <repose-cluster id="repose">
+
+    <nodes>
+      <node id="simple-node" hostname="localhost" http-port="${reposePort}"/>
+    </nodes>
+
+    <filters>
+      <filter name="client-auth"/>
+      <!--<filter name="api-validator"/>-->
```
Comment (dkowis) [inline repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/filters/valkyrie/system-model.cfg.xml]: If we're not going to use the API-validator stuff, we should take out the files...

File: repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/TestProperties.groovy [hunk 2]
```diff
@@ -121,6 +123,7 @@ class TestProperties {
                 configDirectory          : configDirectory,
                 'repose.config.directory': configDirectory,
                 'project.build.directory': projectBuildDirectory,
+                'valkyriePort'           : valkyriePort,
```
Comment (dkowis) [inline repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/TestProperties.groovy]: It's super awesome that we have to put this in three places ....

**Comment by dkowis** [comment]
Looks good, can merge upon successful build.

**Comment by tylerroyal** [comment]
Retest this please

**Pull request merged**